tests/libvmi/factory.go

```diff
@@ -85,6 +85,23 @@ func NewAlpineWithTestTooling(opts ...Option) *kvirtv1.VirtualMachineInstance {
 	return New(alpineOpts...)
 }
 
+func NewGuestless(opts ...Option) *kvirtv1.VirtualMachineInstance {
+	opts = append(
+		[]Option{WithResourceMemory(qemuMinimumMemory())},
+		opts...)
+	return New(opts...)
+}
+
+func qemuMinimumMemory() string {
+	if isARM64() {
+		// required to start qemu on ARM with UEFI firmware
+		// https://github.com/kubevirt/kubevirt/pull/11366#issuecomment-1970247448
+		const armMinimalBootableMemory = "128Mi"
+		return armMinimalBootableMemory
```

Comment:
What is the benefit of defining that constant, whose scope is inside the function, instead of just `return "128Mi"` ?

Reply:
I hate it too. It was a means to silence a linter.